### PR TITLE
add

### PR DESCRIPTION
**Pull request by sakettechnocredit**
Please consider all remaining ooops assigment sorry for the delay

### CODE REVIEW

File: JavaAssignments/src/saket/Addition.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+	void add(float a, float b) {
+		double c = a + b;
+
+		System.out.println(c);
+
+	}
+
+	void add(String a, String b) {
+		String c = a + b;
```
Comment (ghost) [inline JavaAssignments/src/saket/Addition.java:20]: Here it concat the a and b.  It not give me sum 
Right? 
Need to parse value n then you do sum

File: JavaAssignments/src/saket/Areaoverloading.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package saket;
+
+public class Areaoverloading {
+
+	public static void main(String[] args) {
+
+		Area(6,3);
+
```
Comment (ghost) [inline JavaAssignments/src/saket/Areaoverloading.java:8]: Try to follow best practices 
Like method name 
Variables name

File: JavaAssignments/src/saket/Bank.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+import java.util.Scanner;
+
+public class Bank {
+
+	static int takeuserinput() {
+
+		Scanner sc = new Scanner(System.in);
+		Scanner sc1 = new Scanner(System.in);
+		Scanner sc2 = new Scanner(System.in);
```
Comment (ghost) [inline JavaAssignments/src/saket/Bank.java:11]: Here what is need of three scanner class only one is sufficient to get user input

**Review by ghost — APPROVED** [review]
Try to work on the comments 
Logic is correct! 
Keep  practicing

**Review by mkanani13 — APPROVED** [review]

**Pull request merged**